.github/workflows/release-build-publish.yml

```diff
@@ -0,0 +1,256 @@
+name: Release Build Publish
+
+on:
+    release:
+      types: [ published ]
+
+jobs:
+  extract_project_name:
+    runs-on: ubuntu-latest
+    outputs:
+      cleanlab_package_name: ${{ steps.extract_name.outputs.value }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: TOML Reader
+        uses: SebRollen/toml-action@v1.2.0
+        id: extract_name
+        with:
+          file: "pyproject.toml"
+          field: "project.name"
```

Comment:
For clarity. 

This job's `cleanlab_package_name` output should be `cleanlab` within this repo.

Any fork that runs this job cannot upload to https://pypi.org/project/cleanlab/, so a new PyPI project name needs to be configured in the `pyproject.toml`.

This is alluded to in the DEVELOPMENT.md file, where the local prerequisites are mentioned.